### ISSUE TITLE
fix installation paths for case-sensitive file system - fuse v1.2.1 build 13974

### ISSUE THREAD

**Issue by igorls** [fusetools/fuselibs-public#387]
Running a blank app on a freshly installed build 13974 on macOS Sierra 10.12.6 throws this set of errors:
```
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1810): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-BoldItalic.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1806): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-MediumItalic.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1802): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-Italic.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1798): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-LightItalic.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1794): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-ThinItalic.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1789): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-Bold.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1785): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-Medium.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1781): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-Regular.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1777): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-Light.ttf does not exist
/usr/local/share/uno/Packages/Fuse.Common/1.2.1/$.uno(1773): E0000: File /usr/local/share/uno/Packages/Fuse.Common/1.2.1/Internal/DesktopFonts/Roboto-Thin.ttf does not exist
```

**Comment by yupferris** [comment]
Was this a preview app built on another machine? Looks like the font files weren't bundled. Which build target was this?

**Comment by igorls** [comment]
fuse preview (desktop - OSX) on a blank app
<App>
</App>

**Comment by igorls** [comment]
on Fuse.Common/1.2.1/$.uno(1810) the path is defined as Internal (uppercase I), but the installer creates a folder with lowercase i

```
[UXGlobalResource]
		/** The default font of the system, in it's thin weight typeface. */
		extern (!iOS && !Android) public static readonly Font Thin = new Font(import BundleFile("Internal/DesktopFonts/Roboto-Thin.ttf"));
```

**Comment by igorls** [comment]
@yupferris as this is directly related to a case-sensitive file system, I will rename the issue

**Comment by kusma** [comment]
@igorls: Are you by any chance using HFS+ in the non-default case-sensitive setting?

**Comment by igorls** [comment]
@kusma indeed I was (accidentally) something must had gone wrong when I first formatted the SSD, I had no reason to choose that. I've switched to default case-insensitive now and everything is fine. So I suppose that's nothing critical anymore

**Comment by kusma** [comment]
OK, so this looks to me like some parts of the code trying to normalize the casing and other parts not. That's not great ;)

**Comment by mortend** [comment]
This is fixed in an upcoming Uno release (v1.3).
